tests/plugins/clipboard/paste.js

```diff
@@ -808,6 +808,27 @@
 			} );
 		},
 
+		// (#1321)
+		'htmlified text unification 7 - IDEOGRAPHIC space': function() {
+			if ( CKEDITOR.env.gecko )
```

Comment:
Note that our styleguide requires curly braces for every `if` block, even if it contains only one statement.